exercises/alphametics/Hints.md

```diff
@@ -0,0 +1,5 @@
+## Hints
+- To parse the text, you could try to use the [Sprache](https://github.com/sprache/Sprache/blob/develop/README.md) library. 
+There is also good tutorial located [here] (https://www.thomaslevesque.com/2017/02/23/easy-text-parsing-in-c-with-sprache/).
```

Comment:
"There is also good tutorial located [here]" should be "You can find a good tutorial [here]"